dcicutils/s3_utils.py

```diff
@@ -293,6 +293,61 @@ def does_key_exist(self, key, bucket=None, print_error=True):
             return False
         return file_metadata
 
+    def get_object_tags(self, key, bucket):
```

Comment:
Fine with me. I can also change the order of the arguments

Reply:
It might be useful to do both. The order doesn't matter a lot when you're required to tag them, but visually I think people expect bucket to precede so as a purely aesthetic thing I'd do both. You add the "*" with a button press above. To make the other change, you'll need to do editing. :)